dask/array/percentile.py

```diff
@@ -47,21 +47,15 @@ def _percentile(a, q, method="linear"):
 
 
 def _tdigest_chunk(a):
-    from crick import TDigest
-
-    t = TDigest()
-    t.update(a)
-
-    return t
+    return a
 
 
 def _percentiles_from_tdigest(qs, digests):
-    from crick import TDigest
-
-    t = TDigest()
-    t.merge(*digests)
+    import pyarrow as pa
+    import pyarrow.compute as pc
 
-    return np.array(t.quantile(qs / 100.0))
+    tdigest = pc.tdigest(pa.chunked_array(digests), q=(qs / 100.0))
+    return tdigest.to_numpy(zero_copy_only=False)
```

Comment:
I don't think this is going to work. `_tdigest_chunk` is supposed to aggregate the data per chunk. If we're just passing the chunk through, we're effectively collecting all the data on the reducing function. This is almost as costly if we were computing an exact result.